angular-date-picker.js

```diff
@@ -133,7 +135,14 @@
 
                 $scope.pickDay = function (evt) {
                     var target = angular.element(evt.target);
-
+                    
+                    if (target.hasClass('-leadingDay')) {
```

Comment:
The BEM-like convention that used elsewhere in the code would dictate that this can be just named "-leading" and the other "-trailing".